Makefile.dep

```diff
@@ -122,7 +122,7 @@ ifneq (,$(filter gnrc_rpl_p2p,$(USEMODULE)))
 endif
 
 ifneq (,$(filter gnrc_rpl,$(USEMODULE)))
-  USEMODULE += gnrc_ipv6_router_default
+  USEMODULE += gnrc_icmpv6
```

Comment:
Doesn't this change remove the dependency to nib? We should explicitely state the nib dependency here ..

Reply:
You mean the *transitive* dependency? That's still given via `gnrc_icmpv6`. But you are right. I will piggy-back a explicit dependency to `gnrc_ipv6_nib` here.

Reply:
Done